doc/whatsnew/fragments/8595.false_negative

```diff
@@ -0,0 +1,3 @@
+Fix false negative regression caused by #8441.
+
+Closes #8595
```

Comment:
Unreleased bugfix, so we can avoid changelog clutter.

Reply:
Gotcha, removed the changelog. Thanks for the review :)